### DISCUSSION TITLE
Optional software's in ProgramData\Package Cache

### DISCUSSION THREAD

**Discussion by yuvnith** [wixtoolset/issues#7676] [Questions]
Hi, 
In our wix setups, we have an option to launch optional software setups via home page. We provide a relative path of these additional software's and use the same to launch the optional software setup. 

eg:
Main setup -> D:\XYZSoftware\Setup.exe
Optional Software -> D:\XYZSoftware\OptionalSoftwares\Setup2.exe

We observed that a windows installer setup is cached onto "ProgramData\Package Cache" during installation, the same setup.exe would be launched when the user launches the setup again after installation. We would like to cache our additional software setups too in the same location, so that when the user launches the main setup again after installation, the optional software exe's would also be present and can be launched (as relative paths are used) through main setup.

Could someone kindly offer advice on how to implement this functionality?

**Comment by robmen** [comment]
You might try narrowing your question. This is too broad for a Discussion. If you want an expert to dig into the problem with you, check out a [FireGiant phone consultation](https://www.firegiant.com/services/phone-consultation/).

**Reply by yuvnith** [reply]
Thank you for the suggestion. I will reach out to FireGiant
